.github/workflows/ci.yaml

```
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: IN-CORE/pyincore-data
+
+jobs:
+
+  # ----------------------------------------------------------------------
+  # DOCKER BUILD
+  # ----------------------------------------------------------------------
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      # checkout source code
+      - uses: actions/checkout@v2
+
+      # calculate some variables that are used later
+      - name: version information
+        run: |
+          if [ "${{ github.event.release.target_commitish }}" != "" ]; then
+            BRANCH="${{ github.event.release.target_commitish }}"
+          elif [[ $GITHUB_REF =~ pull ]]; then
+            BRANCH="$(echo $GITHUB_REF | sed 's#refs/pull/\([0-9]*\)/merge#PR-\1#')"
+          else
+            BRANCH=${GITHUB_REF##*/}
+          fi
+          echo "GITHUB_BRANCH=${BRANCH}" >> $GITHUB_ENV
+          if [ "$BRANCH" == "master" ]; then
+            version=$(cat package.json | grep \"version\" | head -1 | awk -F= "{ print $2 }" | sed 's/[version:,",]//g' | tr -d '[[:space:]]')
```

I think this should be:
```suggestion
            version=$(awk -F= '/^release/ { print $2}' docs/source/conf.py | sed "s/[ ']//g")
```

---

Changed. I guess that I have to change the same thing to pyincore and pyincore-viz repositories as well since they also uses conf.py

---

remember that the code runs in the root of the git folder, so you can just copy/paste the commands to see what the result is.